print('start')

try:
    print('try is started')
    print(a)
    print('try has ended')

except NameError as ne:
    print(ne)

print(end)

